### DISCUSSION TITLE
Python not found :(

### DISCUSSION THREAD

**Discussion by InfinityX12** [atom/atom#22761] [Q&A]
I'm new to python, as python is my first language.  So I started using pycharm but atom is lighter. I have also installed python 3.9 before doing this. So i made a simple  test file, and installed the script plugin so I could run it. I have also chosen the language as python. For some reason Its saying this. 

![image](https://user-images.githubusercontent.com/87969565/126953811-bfafc9e9-ea11-425f-98cb-927b6c6c9ee2.png)

Please help me (PLS SEND HELP FAST, LMAO)

**Comment by Henriquecesar139** [comment]
Chances are you didn't add python to the 'path'.
You can reinstall python and check the 'add python to path' checkbox, or create a path variable in the ​​environment variables tab
